core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/v2/InvokerHealthManager.scala

```diff
@@ -269,6 +270,7 @@ object InvokerHealthManager {
   val healthActionNamePrefix = "invokerHealthTestAction"
   val bufferSize = 10
   val bufferErrorTolerance = 3
+  var useMemory = 0l
```

Comment:
Why do we need this mutable variable?

Reply:
There has a get used memory API: http://$invoker:$port/memory, 
```
complete(InvokerHealthManager.useMemory.toString)
```
The FunctionPullingContainerPool actor(which will be contributed) sends the pool's used memory info(busyMemory + inprogressMemory) to InvokerHealthyManager actor, so use a mutable variable to store the value.

Reply:
I know that's there only one instance of the invoker health manager per instance / jvm, but it still seems risky to be mutating a var in an object like this especially when it's the source of truth on whether the invoker is idle or not.

Reply:
@style95 @bdoyle0182 , yes, not a good idea of `use mutable variable`
It is better to do like below
```
functionPullingContainerPool.ask(UsedMemory)(Timeout(5.seconds)).mapTo[Int]
```
But functionPullingContainerPool is implementing now, so i removed /memory api in this pr and i will add /memory api in ` [New Scheduler] FunctionPullingContainerPool` pr.